docs/source/interactive/shell.rst

```diff
@@ -186,13 +186,11 @@ First, capture output of "hg status"::
 
 Now we can just remove these files by doing 'rm $junk.s'.
 
-The .s, .n, .p properties
+The .n, .s, .p properties
 -------------------------
```

Comment:
It would be good to have a summary explaining that these provide convenient ways to use the text in different formats.

Reply:
Right. Added.

Reply:
What are all places where SList object is returned from? `returned by execution operator !` is one place, but are there other uses?

Reply:
I'd like to add reference to SList API here.

Reply:
That's the only one I know of; you can search the codebase to see if it's used elsewhere.

Good idea to link to the reference.